src/server/game/Tools/PlayerDump.cpp

```diff
@@ -259,13 +259,14 @@ bool PlayerDumpWriter::DumpTable(std::string& dump, ObjectGuid::LowType guid, ch
 
     switch (type)
     {
-        case DTT_ITEM:      fieldname = "guid";      guids = &items; break;
-        case DTT_ITEM_GIFT: fieldname = "item_guid"; guids = &items; break;
-        case DTT_PET:       fieldname = "owner";                     break;
-        case DTT_PET_TABLE: fieldname = "guid";      guids = &pets;  break;
-        case DTT_MAIL:      fieldname = "receiver";                  break;
-        case DTT_MAIL_ITEM: fieldname = "mail_id";   guids = &mails; break;
-        default:            fieldname = "guid";                      break;
+        case DTT_ITEM:      fieldname = "guid";                guids = &items; break;
+        case DTT_ITEM_GIFT: fieldname = "item_guid";           guids = &items; break;
+        case DTT_PET:       fieldname = "owner";                               break;
+        case DTT_PET_TABLE: fieldname = "guid";                guids = &pets;  break;
+        case DTT_MAIL:      fieldname = "receiver";                            break;
+        case DTT_MAIL_ITEM: fieldname = "mail_id";             guids = &mails; break;
+        case DTT_CURRENCY:  fieldname = "CharacterGuid";                       break;
+        default:            fieldname = "guid";                                break;
```

Comment:
Don't change indentation for the sake of changing indentation. It makes it hard to see real changes.